package.json

```diff
@@ -44,9 +45,14 @@
   "homepage": "https://github.com/4ver/node-auto-launch",
   "dependencies": {
     "applescript": "^1.0.0",
+    "gulp-rimraf": "^1.0.0",
```

Comment:
Should be in dev dependencies

Reply:
fixed by https://github.com/Oxalin/node-auto-launch/commit/e46e9092cf84d0c06570bbcc621bb96358c67290